Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.17 as build
```

Comment:
do they have the `golang:1.18beta1-slim` verions? if not , can go with 1.17

Reply:
`golang:1.18beta1-slim` doesn't exist, let's use 1.17 for now. I can update it when `golang:1.18beta1-slim` is available.